chapters/10-code-navigation.md

```diff
@@ -1,16 +1,16 @@
 # Code Navigation
 
 ## LSP
-To make the nvim "smart" you need to install a lsp server and connect it to nvim's lsp client.
+To make nvim "smart" you need to install a LSP server and connect it to nvim's LSP client.
 
 There are two options to do that:
-1. Using [mason.nvim](https://github.com/williamboman/mason.nvim) with [mason-lspconfig.nvim](https://github.com/williamboman/mason-lspconfig.nvim) and [nvim-lspconfig](https://github.com/neovim/nvim-lspconfig), these 3 plugins will manage and install lsps, make sure you integrate it with your [preconfigured configuration](https://nvchad.github.io/config/Lsp%20stuff).
-1. Using [nvim-lspconfig](https://github.com/neovim/nvim-lspconfig) and manually maintain the lsp servers binaries (I do it personally with dotfiles), make sure you integrate it with your [preconfigured configuration](https://nvchad.github.io/config/Lsp%20stuff).
+1. Using [mason.nvim](https://github.com/williamboman/mason.nvim) with [mason-lspconfig.nvim](https://github.com/williamboman/mason-lspconfig.nvim) and [nvim-lspconfig](https://github.com/neovim/nvim-lspconfig), these 3 plugins will manage and install LSP servers, make sure you integrate it with your [preconfigured configuration](https://nvchad.github.io/config/Lsp%20stuff).
```

Comment:
@ofirgall perhaps we should remove/replace nvchad links since we switched most of them to lvim, and these are broken anyways..